todo_app/app.py

```diff
@@ -6,19 +6,41 @@
 from todo_app.app_config import Config
 from todo_app.viewmodel import ViewModel
 from todo_app.data.user_role import writer_required, isWriterRole
-from flask import Flask, render_template, request, redirect, url_for, g
+from flask import Flask, render_template, request, redirect, url_for, g, has_request_context
 from oauthlib.oauth2 import WebApplicationClient
 import requests
 from todo_app.data.TodoUser import ToDoUser
 from werkzeug.debug import DebuggedApplication
 import todo_app.data.mongo_constants as constants
 from flask_login import current_user
+from flask.logging import default_handler
+import logging 
+from loggly.handlers import HTTPSHandler
+from logging import Formatter
+from pythonjsonlogger import jsonlogger
 
 def create_app():
+    loggingLevel = ""
+    if os.getenv('LOGGING_LEVEL'):
+        loggingLevel = os.getenv('LOGGING_LEVEL').upper()
+    log_level_info = {'DEBUG': logging.DEBUG, 
+                    'INFO': logging.INFO,
+                    'WARNING': logging.WARNING,
+                    'ERROR': logging.ERROR,
+                    }
```

Comment:
Nitpick, but `log_level_info` is a slightly unfortunate variable name since `info` is itself a log level. Maybe `log_level_map`?

Also I think this formatting is slightly nicer
```suggestion
    log_level_info = {
        'DEBUG': logging.DEBUG, 
        'INFO': logging.INFO,
        'WARNING': logging.WARNING,
        'ERROR': logging.ERROR,
    }
```

Reply:
Will change in my module 14 which is nearing its completion.